packages/react-select/src/Select.js

```diff
@@ -1426,6 +1436,13 @@ export default class Select extends Component<Props, State> {
       'aria-labelledby': this.props['aria-labelledby'],
     };
 
+    // Helper function to form aria attributes from aria object prop
+    const attributesFromObject = (prefix, items) =>
+      Object.keys(items).reduce((acc, name) => {
+        acc[`${prefix}-${name}`] = items[name];
```

Comment:
To be honest, this is not straightforward API. For me, I would like to pass the aria attributes to the component directly
```jsx
<Select aria-describedby="sth" />
```
not via separate property. Also, now there are 2 different ways to provide the aria props. And this is not tested nor documented too.

Reply:
So the reason to go this route (aria prop) was to make it generic for adding aria props, so that even in future, if need to add few more aria props, we do not have to open another pull request. This is how `react-modal` is currently supporting for adding aria props. 

And I just did not want to remove the existing aria props because of backward compatibility.

Reply:
I don't think we would want to expose every possible aria attribute.

For example, look at the [list of aria attributes in @types/react](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/index.d.ts#L1558). Not all of these make sense and can lead to even more broken keyboard accessibility for people who have best intentions.

Instead, we should indeed limit it to just the attributes that make sense as the value input for react-select.